lib/rucio/web/rest/flaskapi/v1/replicas.py

```diff
@@ -892,6 +894,46 @@ def post(self):
         return 'Created', 201
 
 
+class QuarantineReplicas(ErrorHandlingMethodView):
+
+    def post(self):
```

Comment:
Before I forget, the documentation for the REST endpoints needs to be written a little bit different now. See
https://github.com/rucio/rucio/blob/2aedf7f83cdb88f5f47a81a6ee29588afad13e6b/lib/rucio/web/rest/flaskapi/v1/dids.py#L373
The reason for this is so it is parsable by our now rest endpoint doc: http://rucio.cern.ch/documentation/html/rest_api_doc.html
Please check with @joeldierkes (on slack?) if you have any questions on it.

Reply:
Yes, feel free to write me if you have any questions. Other files like [rses.py](https://github.com/rucio/rucio/blob/master/lib/rucio/web/rest/flaskapi/v1/rses.py) already implement these style changes.